nodebuilder/share/module.go

```diff
@@ -47,6 +48,20 @@ func ConstructModule(tp node.Type, cfg *Config, options ...fx.Option) fx.Option
 		return fx.Module(
 			"share",
 			baseComponents,
+			fx.Provide(fx.Annotate(
+				func(path node.StorePath, ds datastore.Batching) (*eds.Store, error) {
+					return eds.NewStore(string(path), ds)
+				},
+				fx.OnStart(func(ctx context.Context, eds *eds.Store) error {
+					return eds.Start(ctx)
```

Comment:
short lived context passed down into long-lived op

Reply:
the context that is passed is not used

Reply:
Sir.

Reply:
oops, discussed in sync, was looking at wrong thing